nix/nixos/smash-service.nix

```diff
@@ -69,22 +90,32 @@ in {
     };
   };
   config = lib.mkIf cfg.enable {
-    services.smash.script = pkgs.writeShellScript "smash" ''
+    services.smash.script = let
+    in pkgs.writeShellScript "smash" ''
       set -euo pipefail
 
-      cp ${cfg.postgres.pgpass} $RUNTIME_DIRECTORY/pgpass
+      RUNTIME_DIRECTORY=''${RUNTIME_DIRECTORY:-$(pwd)}
```

Comment:
Added extra logic to handle pgpass being in current directory by defaults when ran with script